easybuild/easyconfigs/b/BWA/BWA-0.7.13-foss-2015b.eb

```diff
@@ -0,0 +1,32 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+#
+# Copyright:: Copyright 2012-2014 Cyprus Institute / CaSToRC, Uni.Lu/LCSB, NTUA
+# Authors::   George Tsouloupas <g.tsouloupas@cyi.ac.cy>, Fotis Georgatos <fotis@cern.ch>
+# License::   MIT/GPL
+# $Id$
+#
+# This work implements a part of the HPCBIOS project and is a component of the policy:
+# http://hpcbios.readthedocs.org/en/latest/HPCBIOS_2012-94.html
+# 
+# Version >= 0.7.13
+# Author: Adam Huffman
+# The Francis Crick Institute 
+##
+
+name = 'BWA'
+version = '0.7.13'
+
+homepage = 'https://github.com/lh3/bwa'
+description = """Burrows-Wheeler Aligner (BWA) is an efficient program that aligns
+ relatively short nucleotide sequences against a long reference sequence such as the human genome."""
+
+toolchain = {'name': 'foss', 'version': '2015b'}
+toolchainopts = {'optarch': True, 'pic': True}
+
+source_urls = ['https://github.com/lh3/%(name)s/archive/']
```

Comment:
See above - all the focus appears to be on GitHub.

(lh3 is Heng Li, developer of BWA)